flight/flight-sql-jdbc-core/pom.xml

```diff
@@ -47,6 +47,21 @@ under the License.
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-api</artifactId>
```

Comment:
I'm surprised we have to explicitly define these dependencies, I thought they came transitively from arrow flight.